### PR TITLE
remove gitlab-ci

### PR DESCRIPTION
**Pull request by vlukashenko**
I removed gitlab-CI mentioning and only left the dockerhub + github. I understood that dockerhub can be used also with podman. 
@michmx

### CODE REVIEW

File: _episodes/08-gitlab-ci.md [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
 teaching: 20
 exercises: 25
 questions:
-- "How can gitlab CI and docker work together to automatically preserve my analysis environment?"
-- "What do I need to add to my gitlab repo(s) to enable this automated environment preservation?"
+- "What do I need to do to enable this automated environment preservation on github?"
 objectives:
-- "Learn how to write a Dockerfile to containerize your analysis code and environment."
-- "Understand what needs to be added to your `.gitlab-ci.yml` file to keep the containerized environment continuously up to date for your repo."
+- "Learn how to werite a Dockerfile to containerize your analysis code and environment."
```
Comment (michmx) [inline _episodes/08-gitlab-ci.md]:
```suggestion
- "Learn how to write a Dockerfile to containerize your analysis code and environment."
```

File: _episodes/08-gitlab-ci.md [hunk 2]
```diff
@@ -213,12 +135,17 @@ docker pull <username>/<image name>:<tag>
 ~~~
 {: .source}
 
+> ## Tag your docker image
+> Notice that the command above had a ``<tag>`` specified. A tag uniquely identifies a docker image. When puched to
```
Comment (michmx) [inline _episodes/08-gitlab-ci.md]: This looks incomplete. A missing commit perhaps?

**Review by michmx — CHANGES_REQUESTED** [review]
Thanks! In addition to couple of comments above, there is a reference left to Gitlab that must be updated (see https://github.com/hsf-training/hsf-training-docker/pull/83/files#diff-3410d82f657b25582e7d2050477b4a1ab76d7ced14d3e8b84c80158371fcddcdR69).

**Comment by mdonadoni** [comment]
FYI DockerHub automated builds are now a paid feature (see https://docs.docker.com/docker-hub/builds/ and https://forums.docker.com/t/linked-accounts-tab-missing/111606), so I actually removed that from the lesson for APB in Valencia (see https://github.com/hsf-training/hsf-training-docker/pull/84). Also note that the lesson was focusing mostly on GitLab CI as that's what is used at CERN.

File: _episodes/08-gitlab-ci.md (hunk 2 above)
Comment (vlukashenko) [reply _episodes/08-gitlab-ci.md]: Yeah this is finishing on monday evening and starting again on tuesday morning :D

**Review by michmx — APPROVED** [review]
Looks good! Today and tomorrow I will go ahead with the switch to Podman. Likely a few things will have to be updated (I will add you in the pull request).

**Pull request merged**